Jetcaster/settings.gradle.kts

```diff
@@ -35,5 +35,5 @@ dependencyResolutionManagement {
     }
 }
 rootProject.name = "Jetcaster"
-include(":mobile", ":core", ":core:model", ":designsystem", ":tv-app", ":wear")
+include(":mobile", ":core", ":core:model", ":designsystem", ":tv-app", ":wear", ":testing")
```

Comment:
Can we put testing under `:core`? Similar to the pattern I did in #1382. I would also recommend renaming `:testing` to `:core:domain-testing` since the contained mocks mock domain models.

Reply:
done